package.json

```diff
@@ -54,6 +57,8 @@
     "react-use": "^15.3.4",
     "sharp": "^0.29.1",
     "snyk": "^1.437.4",
+    "source-map-loader": "^3.0.1",
+    "ts-loader": "^9.2.6",
```

Comment:
These should be dev dependencies in a normal react app, please disregard if that's not the case for a Gatsby site.